File: projects/2/jaylon_williams/hello.jl
function main()
    
    if length(ARGS) < 4 || ARGS[1] != "--name" || ARGS[3] != "--number"
        println("Usage: julia hello.jl --name <name> --number <number>")
        return
    end

    name = ARGS[2]
    number = parse(Int, ARGS[4])  

    for _ in 1:number
        println("Hello $name")
    end
end

if abspath(PROGRAM_FILE) == abspath(@__FILE__)
    main()
end

File: projects/3/xavier_baines/projects/2/jon_white/hello.nim
import os, strutils

proc main() =
  if paramCount() < 1:
    echo "Usage: ./hello <name> [number]"
    return

  let name = paramStr(1)
  let count = if paramCount() > 1: parseInt(paramStr(2)) else: 1

  for i in 1..count:
    echo i, ": Hello ", name

main()
